src/js/index.js

```diff
@@ -38,8 +38,8 @@ export const hpe = deepFreeze({
         light: '#FFFFFF',
       },
       'background-contrast': {
-        dark: '#FFFFFF12',
-        light: '#11111108',
+        dark: '#1FFFFFFF',
+        light: '#99CCCCCC',
```

Comment:
This looks like we've swapped the opacity to the first bytes?

Reply:
@ericsoderberghp I was following this convention:  https://gist.github.com/lopspower/03fb1cc0ac9f32ef38f4#how-to-set-transparency-with-hex-value-

Reply:
I believe that's only for Android. Here's the way grommet does it for regular browsers: https://css-tricks.com/8-digit-hex-codes/

Reply:
Thanks for catching that @ericsoderberghp